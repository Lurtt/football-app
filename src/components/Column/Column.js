import styled from 'styled-components'

import { maxWidth, textAlign } from '../mixins'

export default styled.div`
  flex: 1;
  ${maxWidth};
  padding: 1rem;
  ${textAlign};
  overflow: hidden;
`
