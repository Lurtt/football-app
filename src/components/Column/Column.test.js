import React from 'react'
import renderer from 'react-test-renderer'

import Column from './'

describe('<Column />', () => {
  test('Should render', () => {
    const result = renderer
      .create(
        <Column maxWidth="22rem" textAlign="center">
          Content here
        </Column>
      )
      .toJSON()
    expect(result).toMatchSnapshot()
  })
})
