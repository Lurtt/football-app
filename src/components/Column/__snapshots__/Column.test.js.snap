// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> Should render 1`] = `
<div
  className="sc-bdVaJa dzUGjy"
>
  Content here
</div>
`;
